tests/UI/campaigns/functional/BO/11_international/01_localization/currencies/05_filterSortAndPagination.js

```diff
@@ -0,0 +1,304 @@
+      expect(paginationNumber).to.contains('(page 1 / 1)');
+    });
+  });
+
+  // 4 : Sort table
+  describe('Sort currencies table', async () => {
+    const tests = [
+      {
+        args: {
+          testIdentifier: 'sortByIdDesc', sortBy: 'id_currency', sortDirection: 'desc', isFloat: true,
+        },
+      },
+      {args: {testIdentifier: 'sortByIsoCodeAsc', sortBy: 'iso_code', sortDirection: 'asc'}},
+      {args: {testIdentifier: 'sortByIsoCodeDesc', sortBy: 'iso_code', sortDirection: 'desc'}},
+      {
+        args: {
+          testIdentifier: 'sortByExchangeRateAsc', sortBy: 'conversion_rate', sortDirection: 'asc', isFloat: true,
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByExchangeRateDesc', sortBy: 'conversion_rate', sortDirection: 'desc', isFloat: true,
+        },
+      },
+      {args: {testIdentifier: 'sortByEnabledAsc', sortBy: 'active', sortDirection: 'asc'}},
+      {args: {testIdentifier: 'sortByEnabledDesc', sortBy: 'active', sortDirection: 'desc'}},
+      {
+        args: {
+          testIdentifier: 'sortByIdAsc', sortBy: 'id_currency', sortDirection: 'asc', isFloat: true,
+        },
+      },
+    ];
+
+    tests.forEach((test) => {
+      it(`should sort by '${test.args.sortBy}' '${test.args.sortDirection}' and check result`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', test.args.testIdentifier, baseContext);
+
+        let nonSortedTable = await currenciesPage.getAllRowsColumnContent(page, test.args.sortBy);
+
+        await currenciesPage.sortTable(page, test.args.sortBy, test.args.sortDirection);
+
+        let sortedTable = await currenciesPage.getAllRowsColumnContent(page, test.args.sortBy);
+
+        if (test.args.isFloat) {
+          nonSortedTable = await nonSortedTable.map(text => parseFloat(text));
+          sortedTable = await sortedTable.map(text => parseFloat(text));
+        }
+
+        const expectedResult = await currenciesPage.sortArray(nonSortedTable, test.args.isFloat);
+
+        if (test.args.sortDirection === 'asc') {
+          await expect(sortedTable).to.deep.equal(expectedResult);
+        } else {
+          await expect(sortedTable).to.deep.equal(expectedResult.reverse());
+        }
+      });
+    });
+  });
+
+  // 4 : Delete currencies created
+  describe('Delete currencies', async () => {
+    currencies.forEach((currency, index) => {
+      it(`should filter list by currency name '${currency.name}'`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `filterToDelete${index}`, baseContext);
+
+        await currenciesPage.filterTable(page, 'input', 'name', currency.name);
+
+        const currencyName = await currenciesPage.getTextColumnFromTableCurrency(page, 1, 'name');
+        await expect(currencyName).to.contains(currency.name);
+      });
+
+      it('should delete currency', async function () {
```

Comment:
```suggestion
      it(`should delete currency '${currency.name}'`, async function () {
```

Reply:
At the previous step, the message is `should filter the list by currency name '${currency.name}'`, I think it's clear to delete the searched currency. WDYT?

Reply:
Yes it should be enough :+1: